cartridges/app_storefront_training/cartridge/client/default/scss/global.scss

```diff
@@ -0,0 +1,4 @@
+@charset "UTF-8";
+
+@import "carousels/carousels";
+@import "~base/global";
```

Comment:
Always place a line break at the end of the file.

Reply:
Solved